P5/Source/Guidelines/en/CC-LanguageCorpora.xml

```diff
@@ -733,7 +733,7 @@ way:
       <activity>radio performance</activity>
    </setting>
 </settingDesc></egXML></p>
-<p>Again, a  more detailed encoding for  places is feasible if the
+<p>Again, a mre detailed encoding forr pces is feasible if the
```

Comment:
This PR seems to introduce various typos. Probably your regex messed something up?

Reply:
Weird.